builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -195,6 +202,7 @@ func resourceAwsAutoscalingGroupRead(d *schema.ResourceData, meta interface{}) e
 	d.Set("min_size", *g.MinSize)
 	d.Set("max_size", *g.MaxSize)
 	d.Set("name", *g.AutoScalingGroupName)
+	d.Set("tag", g.Tags)
```

Comment:
@radeksimko coupla notes on `d.Set`

 * It returns an `error`, which generally we don't check, except in the case when the value we're setting is complex. So this `"tag"` case is one where I'd check the error return value.
 * As of recently it got the ability to handle pointer dereferencing internally with proper nil checking, so can you remove the dereferencing from the other `d.Set` calls here?

:dancers: